app/views/triage/edit.html.erb

```diff
@@ -1,7 +1,8 @@
 
 <% if @contact_needs.errors.any? || @contact.errors.any? %>
```

Comment:
I assume changes to this file is just formatting/linting/rubocop fixes?

Reply:
Correct, rubymine formatting after I had added and then removed something